src/qtfirebaseremoteconfig.cpp

```diff
@@ -1,4 +1,4 @@
-#include "qtfirebaseremoteconfig.h"
+QtFirebaseAnalytics *QtFirebaseAnalytics::self = 0;#include "qtfirebaseremoteconfig.h"
```

Comment:
Also... You are using `nullptr` [here](https://github.com/Larpon/QtFirebase/pull/18/commits/5a6fddfe46b81764b0e3c164c8378f6b8fe8173d#diff-585a0f1968f92dc12e5893278d4b759dR6) but `0` in this case.